react/components/FacetCheckboxList.js

```diff
@@ -1,37 +1,59 @@
 import classNames from 'classnames'
-import React from 'react'
+import React, { useState, useContext } from 'react'
 import { Checkbox } from 'vtex.styleguide'
 
 import useSelectedFilters from '../hooks/useSelectedFilters'
 import styles from '../searchResult.css'
+import { useSortFacets } from '../constants/SearchHelpers'
+import SettingsContext from './SettingsContext'
+import SearchFilterBar from './SearchFilterBar'
 
-const FacetCheckboxList = ({ facets, onFilterCheck }) => {
+const FacetCheckboxList = ({ facets, onFilterCheck, title }) => {
   const facetsWithSelected = useSelectedFilters(facets)
+  const { orderFacetsBy, showFacetSearch, showFacetQuantity } = useContext(
```

Comment:
please create a `useSettings` that abstracts the usage of `useContext`

Reply:
Done! It is a way better!